xbmc/guilib/GUIListContainer.h

```diff
@@ -26,12 +26,13 @@ class CGUIListContainer : public CGUIBaseContainer
 {
 public:
   CGUIListContainer(int parentID, int controlID, float posX, float posY, float width, float height, ORIENTATION orientation, const CScroller& scroller, int preloadItems);
-//#ifdef GUILIB_PYTHON_COMPATIBILITY
+  explicit CGUIListContainer(const CGUIListContainer& other);
+  //#ifdef GUILIB_PYTHON_COMPATIBILITY
```

Comment:
Not relevant to this PR but this ifdef seems odd, it can probably be removed in the future.

Reply:
Indeed, although we know from the fallacy of Chesterton's Fence that these removals deserve much scrutiny and should be done carefully. Putting in this effort to remove a comment is definitely overkill for me, but maybe someday someone will pick up the torch.